keyboards/gvlsplit/info.json

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
Please run `qmk format-json` on this file.

Reply:
ran `qmk format-json -f keymap info.json`

Reply:
This isn't a keymap.